pkg/daemon/ceph/client/filesystem.go

```diff
@@ -179,69 +179,22 @@ func IsMultiFSEnabled() bool {
 
 // SetNumMDSRanks sets the number of mds ranks (max_mds) for a Ceph filesystem.
 func SetNumMDSRanks(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, fsName string, activeMDSCount int32) error {
```

Comment:
`cephVersion` doesn't seem to be used by the function any more.

Reply:
Thanks for the pointer, apparently my VScode is having some issues and that is why it's not showing unused variables passed in the function params.
Will fix it.